pkg/cmd/drtprod/helpers/utils.go

```diff
@@ -82,5 +77,19 @@ func ExecuteCmd(ctx context.Context, logPrefix string, cmd string, args ...strin
 	}()
 
 	// Wait for the command to complete and return any errors encountered.
-	return c.Wait()
+	return c.Run()
+}
+
+// ExecuteCmdInteractive runs a shell command with the given arguments and creates an interactive shell.
+func ExecuteCmdInteractive(ctx context.Context, cmd string, args ...string) error {
+	// Create a command with the given context and arguments.
+	c := exec.CommandContext(ctx, cmd, args...)
+
+	// redirect stdin, stdout and stderr
+	c.Stdin = os.Stdin
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
```

Comment:
nit: could be one liner `c.Stdin, c.Stdout, c.Stderr = os.Stdin, os.Stdout, os.Stderr`